src/docs/Doxyfile.in

```diff
@@ -3,7 +3,7 @@ GENERATE_LATEX = NO
 INPUT = "@DOXYGEN_INPUT_DIR@"
 OUTPUT_DIRECTORY = "@DOXYGEN_OUTPUT_DIR@"
 PROJECT_BRIEF = "Azure IoT Agent that delivers client update from Azure Delivery Update (ADU)."
```

Comment:
still reads "Azure IoT agent" and "Azure Delivery Update" -- are those correct?

Reply:
Asking esha.